docs/oldabe.scrbl

```diff
@@ -25,6 +32,8 @@ Old Abe considers precisely three inputs in doing all of its accounting, and it
 
 @item{Attributions -- an association of contributor to percentage of value allocated from the value represented by the project as a whole.}
 
+@item{Instruments -- an association of an instrument to percentage of value allocated from the value represented by the project as a whole.}
```

Comment:
For future reference: we could use Scribble's `deftech` to make this a "definition" that could be linked from elsewhere in the docs

Reply:
@jairtrejo suggests: clarify difference between instruments and attributions here